src/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Twitter Clone</title>
+</head>
+
+<body>
+  <div class="grid-container">
+    <!-- Navbar -->
+    <section class="nav-bar">
+      <div class="logo"><img src="src/assets/icons/twitter.svg"></div>
+
+      <div class="navigation">
+        <div class="nav-grid-container">
+          <nav><img src="src/assets/icons/home.svg" class="home">Home</nav>
+          <nav><img src="src/assets/icons/hashtag.svg" class="explore">Explore</nav>
+          <nav><img src="src/assets/icons/bell.svg" class="notifications">Notifications</nav>
+          <nav><img src="src/assets/icons/messages.svg" class="messages">Messages</nav>
+          <nav><img src="src/assets/icons/bookmark.svg" class="bookmarks">Bookmarks</nav>
+          <nav><img src="src/assets/icons/profile.svg" class="profile">Profile</nav>
+          <nav><img src="src/assets/icons/more.svg" class="more">More</nav>
+          <div class="tweet">Tweet</div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Main Content -->
+    <section class="main">
+      <!-- Header -->
+      <div class="grid-container-header">
+        <header class="home">Home</header>
+        <img src="src/assets/icons/sparkle.svg" class="sparkle">
+      </div>
+
+      <!-- Write a Tweet Section -->
+      <div class="write-post">
+        <div class="circle"><img src="src/assets/img/yippie.png" class="profile-pic"></div>
+        <div class="write-post-contents">Whats happening?</div>
+
+        <div class="write-icons">
+          <img src="src/assets/icons/photo.svg">
```

Comment:
You could potentially give more elements classes to select them specifically if needed and apply the same styling if repeated like the images in your posts

Reply:
will do! thank you :)